examples/oci/connect_vcns_using_multiple_vnics/provider.tf

```diff
@@ -1,7 +1,7 @@
 // Copyright (c) 2018, 2019, Oracle and/or its affiliates. All rights reserved
 // Licensed under the Universal Permissive License v 1.0 as shown at http://oss.oracle.com/licenses/upl.
 provider "oci" {
-  version              = ">= 3.14"
+  #version              = ">= 3.14"
```

Comment:
if version isn't needed then remove the line rather than comment it

Reply:
Good catch. It's a miss on my part.

Reply:
Fixed